_includes/v22.1/orchestration/kubernetes-upgrade-cluster-manual.md

```diff
@@ -138,7 +138,7 @@
 
     {% endif %}
 
-1. Run the following SQL query to verify that the number of under-replicated ranges is zero:
+1. Run the following SQL query to verify that the number of underreplicated ranges is zero:
```

Comment:
I understand why this was changed to have the hyphen, but I put it back because it matches the Helm topics and also matches the column name in the output.

Reply:
I don't really agree, but I can let it go for this PR. Can you make a style issue for this? A quick search shows that the word is overwhelmingly hyphenated in our docs. Personally I don't think we should defer to the Helm docs or to the console output here, especially as you're referring to a parameter name to which grammatical style barely applies anyway.

Reply:
https://cockroachlabs.atlassian.net/browse/DOC-6295